submodules/wizard/wizard.js

```diff
@@ -517,7 +517,7 @@ define(function(require) {
 				loadData: function(asyncCallback) {
 					self.wizardGetUserList({
 						data: {
-							accountId: monster.util.isReseller(monster.apps.auth.currentAccount) ? self.accountId : monster.apps.auth.currentAccount.reseller_id,
+							accountId: monster.apps.auth.currentAccount.is_reseller ? self.accountId : monster.apps.auth.currentAccount.reseller_id,
```

Comment:
I changed this, because using `monster.util.isReseller()` was always retuning `true` even though the account wasn't reseller

Reply:
Good catch! The documentation is indeed misleading, [`monster.util#isReseller`](https://github.com/2600hz/monster-ui/blob/master/src/js/lib/monster.util.js#L251) does not take any parameters!

Reply:
I'll create a ticket to support this behavior as it would make sense for this util to be used with different account objects than `monster.apps.auth.originalAccount'`.